test/bacnet/datalink/bvlc/src/main.c

```diff
@@ -820,7 +820,7 @@ static void test_BVLC_BBMD_Address(void)
 {
     uint8_t apdu[480] = { 0 };
     uint16_t apdu_len = 0;
-    uint16_t test_apdu_len = 0;
+    int16_t test_apdu_len = 0;
```

Comment:
bvlc_foreign_device_bbmd_host_address_decode returns signed value (negative value for errors)